inginious/frontend/plugins/grader_generator/pages/multilang_form.py

```diff
@@ -26,7 +26,10 @@ def tests_to_dict(self):
         for key in keys_to_remove:
             del self.task_data[key]
 
+        # Convert the key to int to get a correct sort process
         # Order test cases to load them correctly
+        grader_test_cases = {int(key): val for key, val in grader_test_cases.items()}
+
         grader_test_cases = OrderedDict(sorted(grader_test_cases.items()))
```

Comment:
What if we do this, does it still work?
```suggestion
        grader_test_cases = OrderedDict(sorted(grader_test_cases.items(), key=lambada x: int(x[0])))
```

Reply:
No, invalid syntax

Reply:
The notebook already converts the keys to int. I didn't have to do anything